CHANGELOG.md

```diff
@@ -59,75 +52,44 @@
+* **menu:** To create sub-menus, you can now add MenuItems directly under a parent MenuItem, no need to wrap them in a Menu/FlyoutMenu anymore
+* **core:** Relocate all *Field components to @dhis2/ui-widgets. They can be accessed from `@dhis2/ui` using named exports.
+* **core:** Field has been reimplemented to compose a field control, it now adds the Label, Help, Validation components instead of being a simple div wrapper, which allows us to avoid the code duplication in each *Field component.
+* **core:** ToggleGroup has been removed. Use a FieldSet for grouping form controls.
+* **togglegroupfield:** ToggleGroupField has been renamed to FieldSetField, which adds the necessary Label, Help, and Validation components to an entire group of components.
+* **field:** Field now provides a composition to provide all necessary things for a *Field component.
+* **radiogroup:** RadioGroup has been deleted.
+* **checkboxgroup:** CheckboxGroup has been deleted.
+* **checkboxgroupcontrol:** CheckboxGroupControl has been deleted.
+* **radiogroupcontrol:** RadioGroupControl has been deleted.
 * **forms:** final-form and react-final-form exports are now re-exported under the named exports FinalForm and ReactFinalForm respectively.
-* These new components replace the `Backdrop` and the `ScreenCover`, which had a slightly unclear scope and have now been removed.
-* - SingleSelect selection is now a string instead of an object with a value and label property
-- MultiSelect selection is now an array of strings instead of an array of objects with a value and label property
+* **layering:** We changed the set of components used to produce various types of overlays: `Layer` is an overlay component that fills the entire screen/page. Besides that it is also a key component to stack various components on top of one another. `ComponentCover` is a similar component that only fills its parent, provided that has a non-static position. Both the `Layer` and the `ComponentCover` can be blocking or non-blocking, accept an `onClick` and a `translucent` prop. `CenterContent` is a component that does exactly what it says on the tin. It also has a `position` prop which can be used to vertically align the content at the `top`, `middle` (default), or `bottom`. These new components replace the `Backdrop` and the `ScreenCover`, which had a slightly unclear scope and have now been removed.
+* **singleselect:** SingleSelect selection is now a string instead of an object with a value and label property
+* **multiselect:** MultiSelect selection is now an array of strings instead of an array of objects with a value and label property
 * **core:** base components can hold a ref.
-* All @dhis2/ui-core exports have been migrated to
-@dhis2/ui-widgets.
-* **forms:** Postfix all the @dhis2/ui-forms components with
-'Control' to avoid conflicts with the base components in @dhis2/ui-core
-and @dhis2/ui-widgets, since all components are now exported in
-@dhis2/ui.
```

Comment:
Maybe we should remove this as well, was refactored later.

Reply:
What do you think @HendrikThePendric, should we add a note for the change to the `FieldFF` suffux?

Reply:
Yup adding a note about the FieldFF suffix would be nice

Reply:
Ok, done